tests/FSharp.Compiler.ComponentTests/TypeChecks/Graph/FileContentMappingTests.fs

```diff
@@ -45,10 +45,10 @@ module X.Y.Z
 """
 
     match content with
-    | [ TopLevelNamespace "X.Y" [] ] -> Assert.Pass()
+    | [ TopLevelNamespace "X.Y" [] ] -> ()
```

Comment:
[No](https://stackoverflow.com/a/14631983/3232646) `Assert.Pass()` in Xunit. Just returning from the method does the trick here.